[0m01:55:31.822129 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7bd6ba67a5a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7bd6ba2d4aa0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7bd6bad789b0>]}


============================== 01:55:31.826903 | 51e8a693-ac14-4b4f-b0b4-761c2c4ebb30 ==============================
[0m01:55:31.826903 [info ] [MainThread]: Running with dbt=1.9.3
[0m01:55:31.827356 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m01:55:31.836653 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/home/ubuntu/.dbt')}
[0m01:55:31.837296 [info ] [MainThread]: Creating dbt configuration folder at 
[0m01:56:51.298544 [error] [MainThread]: Encountered an error:

[0m14:01:34.871666 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b311c0a060>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b311d430b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b310279250>]}


============================== 14:01:34.879015 | b491a8cb-8601-432e-80bb-680dcc2a84f9 ==============================
[0m14:01:34.879015 [info ] [MainThread]: Running with dbt=1.9.4
[0m14:01:34.879561 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init gainers', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m14:01:34.926621 [debug] [MainThread]: Starter project path: /home/ubuntu/SP25_DS5111_pvq8hv/scripts/env/lib/python3.12/site-packages/dbt/include/starter_project
[0m14:01:34.937071 [info ] [MainThread]: 
Your new dbt project "gainers" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:01:34.937828 [info ] [MainThread]: Setting up your profile.
[0m14:09:16.217888 [info ] [MainThread]: Profile gainers written to /home/ubuntu/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m14:09:16.220362 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 461.4104, "process_in_blocks": "95888", "process_kernel_time": 0.228246, "process_mem_max_rss": "153960", "process_out_blocks": "80", "process_user_time": 2.043049}
[0m14:09:16.220815 [debug] [MainThread]: Command `dbt init` succeeded at 14:09:16.220702 after 461.41 seconds
[0m14:09:16.221126 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b308ddef30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b307a6bdd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77b307845f10>]}
[0m14:09:16.221503 [debug] [MainThread]: Flushing usage events
[0m14:09:16.235946 [debug] [MainThread]: An error was encountered while trying to flush usage events
